velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -324,19 +416,10 @@ std::shared_ptr<const core::PlanNode> SubstraitVeloxPlanConverter::toVeloxPlan(
 std::shared_ptr<const core::PlanNode> SubstraitVeloxPlanConverter::toVeloxPlan(
     const ::substrait::Plan& sPlan) {
   // Construct the function map based on the Substrait representation.
-  for (const auto& sExtension : sPlan.extensions()) {
-    if (!sExtension.has_extension_function()) {
-      continue;
-    }
-    const auto& sFmap = sExtension.extension_function();
-    auto id = sFmap.function_anchor();
-    auto name = sFmap.name();
-    functionMap_[id] = name;
-  }
+  constructFuncMap(sPlan);
 
-  // Construct the expression converter.
-  exprConverter_ =
-      std::make_shared<SubstraitVeloxExprConverter>(subParser_, functionMap_);
+  // Create the expression converter.
+  exprConverter_ = std::make_shared<SubstraitVeloxExprConverter>(functionMap_);
```

Comment:
`exprConverter_` and `functionMap_` should not be kept as class field if they are recreated and changed with each invocation.

Reply:
The exprConverter_ and functionMap_ will only be created once for a plan converter, so we kept it here for the reuse.